tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -601,3 +601,32 @@ def test_negative_snapshot_create(bin_cloner_path):
     assert not os.path.exists('memfile')
 
     vm.kill()
+
+
+def test_mmap_seccomp_regression(test_microvm_with_api):
```

Comment:
I don't think this test will age well in this form. I'd like us to integrate this test case into an already existing test, maybe functional snapshot tests, and we can mention there what the regression that we're testing for was. I think it will make more sense long term.

Reply:
I personally like the separation between regression tests and other types of tests (functional, negative, etc.). They are also easier to spot given the `@type: regression` adnotation.

Why do you think having this separate would be better in the longer term?

Reply:
As we chatted offline, we renamed the regression test to `test_create_large_diff_snapshot`